server/models/post-comment-model.js

```diff
@@ -12,6 +15,14 @@ exports.insertComment = (id, bodyArg, authorArg) => {
         if (!articleIds.includes(Number(id)) && Number(id)) {
             return Promise.reject({ status:404, msg: "Not Found"})
         }
+        return db.query(`SELECT username FROM users;`)
```

Comment:
username is a foreign key so has constraints on it anyway, you will find if you try to insert with a username that doesnt exist you will get an error from psql which would eliminate your need to handle it manually